tools/checksig-census/context.py

```
@@ -367,31 +361,39 @@ def _require_hex(
     try:
         decoded = bytes.fromhex(value)
     except ValueError:
-        raise ContextError(
-            f"CTX-DIAG line {line_number}: {field} must be lowercase hex"
-        )
+        raise ContextError(f"CTX-DIAG line {line_number}: {field} must be lowercase hex")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)